dbcon/execplan/rewrites.cpp

```diff
@@ -83,6 +80,13 @@ enum class ChildType
   Leave
 };
 
+enum class GoTo
+{
```

Comment:
Is it possible to re-use GoTo enum defined in parstree.h?

Reply:
Yeah, this was created after, but maybe now it makes sense to reuse.